### ISSUE TITLE
The website domain is blocked by NIXI.

### ISSUE THREAD

**Issue by tusharhero** [gnulinuxindia/site#7]
the website domain [gnulinuxindia.in](gnulinuxindia.in) is being blocked by NIXI.

**Comment by gtlsgamr** [comment]
UPDATE: Looks like nixi.in has put our domain on hold right now. I am not sure when it will get lifted. Here is the email communication so far.


```
From: Hittarth Thummar <hittarth91@gmail.com>
Subject: Request for Review and Removal of Server Hold Status for gnulinuxindia.in
Date: 27 April 2024 at 5:36:49 PM IST
To: legal@nixi.in, registry@nixi.in

Dear NIXI Team,

I hope this message finds you well. I am writing to seek your assistance regarding an urgent issue affecting my domain, gnulinuxindia.in, which is currently under a Server Hold status.

It has come to my attention through my registrar, Namecheap, that the hold on the domain was implemented by your registry. I understand that domains may be placed on hold due to involvement in abusive operations; however, I am unaware of any such activities associated with my domain that would warrant this action.

Could you please provide specific details regarding the reason for the Server Hold? Additionally, I would appreciate guidance on the necessary steps to resolve this matter promptly.

Thank you for your attention to this urgent issue. I look forward to your prompt response to clarify and rectify the situation.

Best regards,

Hitarth Thummar
```

**Comment by gtlsgamr** [comment]
Had a call with NIXI customer care support and they told me that unless I submit my KYC verification with the same address as WHOIS record, they will not unblock it. We switched to https://gnulinuxindia.sh domain.
